packages/landingpage/README.md

```diff
@@ -0,0 +1,17 @@
+# NFT Passports Landing Page
+
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app). It leads to all of the marketing materials for the app, which includes the landing page, documentation, blog, etc.
```

Comment:
Took out some of the create-next-app readme boilerplate and added a sentence/title specific to each project